diesel_cli/src/print_schema.rs

```diff
@@ -42,6 +43,24 @@ impl Default for DocConfig {
     }
 }
 
+/// How to group tables in `allow_tables_to_appear_in_same_query!()`.
+#[derive(Debug, Deserialize, Serialize, Clone, Copy)]
+pub enum AllowTablesInSameQuery {
+    /// Group by foreign key relations
+    #[serde(rename = "fk_related_tables")]
+    FkRelatedTables,
+    /// List all tables in invocation
+    #[serde(rename = "all_tables")]
+    AllTables,
+}
+
+#[allow(clippy::derivable_impls)] // that's not supported on rust 1.65
+impl Default for AllowTablesInSameQuery {
+    fn default() -> Self {
+        AllowTablesInSameQuery::AllTables
+    }
+}
```

Comment:
We should be fine to use the derive now, as our minimal supported rust version is currently 1.78

Reply:
Looks fine to me to remove it here yes. Thanks! 😊

Reply:
Ah I think the comment was to not just remove the comment but actually use the derive on these types, since now we can